viewer/src/public/RevealManagerBase.ts

```diff
@@ -72,21 +114,191 @@ export class RevealManagerBase<TModelIdentifier> implements RenderManager {
     }
   }
 
+  public get renderMode(): RenderMode {
```

Comment:
I will accept this for now, I do wish to have a talk about the consequences of having a single point of entry.

Reply:
Node that I did not add `implements RenderManager` to `RevealManagerBase`. As long as `RevealManagerBase` actually implements that I can't see why `RenderMode` doesn't belong there...